content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -8,21 +8,21 @@ weight: 40
 ---
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+{{< feature-state feature_gate_name="VolumeAttributesClass" >}}
 
 This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
-[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+[Volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
 in Kubernetes.
 
 <!-- body -->
 
 A VolumeAttributesClass provides a way for administrators to describe the mutable
 "classes" of storage they offer. Different classes might map to different quality-of-service levels.
-Kubernetes itself is unopinionated about what these classes represent.
+Kubernetes itself is un-opinionated about what these classes represent.
 
-This is an alpha feature and disabled by default.
+This is a beta feature and disabled by default.
 
-If you want to test the feature whilst it's alpha, you need to enable the `VolumeAttributesClass`
+If you want to test the feature whilst it's beta, you need to enable the `VolumeAttributesClass`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the kube-controller-manager and the kube-apiserver. You use the `--feature-gates` command line argument:
 
```

Comment:
Do you also need to enable any API group?

Reply:
Yes, what would be good phrasing for this?

"Enable the `storage.k8s.io/v1beta1` API group via  `kube-apiserver` [runtime-config](https://kubernetes.io/docs/tasks/administer-cluster/enable-disable-api/)"

Reply:
Added in latest commit.